pubobot/bot.py

```diff
@@ -878,11 +880,11 @@ async def processmsg(self, msg):
             elif lower[0] == "unsubscribe":
                 await self.subscribe(member, lower[1:msglen], True)
 
-            elif lower[0] == "lastgame":
-                self.lastgame(member, msgtup[1:msglen])
-
-            elif lower[0] == "last":
-                self.lastgame(member, msgtup[1:msglen])
+            elif last_match:
+                t_count = len(last_match.group(1))
```

Comment:
This is a great candidate for the walrus operator.
```suggestion
            elif (last_match := re.match(r"last(t*)", lower[0])):
                t_count = len(last_match.group(1))
```
That way the regex pattern is localized and the assignment above removed.

Reply:
thanks, was not sure how to do that